grounding/src/main/java/com/sap/ai/sdk/grounding/GroundingClient.java

```diff
@@ -0,0 +1,70 @@
+package com.sap.ai.sdk.grounding;
+
+import com.sap.ai.sdk.core.AiCoreService;
+import com.sap.ai.sdk.grounding.api.PipelinesApi;
+import com.sap.ai.sdk.grounding.api.RetrievalApi;
+import com.sap.ai.sdk.grounding.api.VectorApi;
+import javax.annotation.Nonnull;
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import lombok.experimental.Tolerate;
+
+/**
+ * Service class for the Grounding APIs.
+ *
+ * @since 1.3.0
+ */
+@RequiredArgsConstructor(access = AccessLevel.PUBLIC)
+@Getter(value = AccessLevel.PROTECTED)
+public class GroundingClient {
```

Comment:
Could we possibly remove this class the same way we removed it for AI Core:
```java
ArtifactApi() {
    super(new AiCoreService().getApiClient());
  }
```

Reply:
I wish I can do it. Unfortunately, I'm not aware how to propagate the default path `"lm/document-grounding/"`. Do you have an idea how to consider this?

Reply:
I thought of a generator update and a parameter in getApiClient().
But on second thought, This class would allow us to add mix-ins and is more flexible.